spec/models/podcast_episode_spec.rb

```diff
@@ -9,6 +9,7 @@
 
       it { is_expected.to belong_to(:podcast) }
       it { is_expected.to have_many(:comments).inverse_of(:commentable).dependent(:nullify) }
+      it { is_expected.to have_many(:podcast_episode_appearances).dependent(:destroy) }
```

Comment:
this is missing the check for `  has_many :users, through: :podcast_episode_appearances` I think

Reply:
can you address this as well @NenadPantelic?

Reply:
@rhymes I've done that

Reply:
sorry, didn't see it! Thanks!